### PR TITLE
Fix #712: Update Profile Photo Selection #708

### LINKED ISSUE #712: Lowfi: Updated picture selection
Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/62215bbd-1f4e-448e-b370-19c4a06fa814/PC-MP-Add-Profile-PINs-Match-Error-1

Current:
Screenshot 2020-02-25 at 4 22 35 PM

New as per mock:
Screenshot 2020-02-25 at 4 22 42 PM

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
## Explanation

Screenshot 2020-02-25 at 5 40 03 PM

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/drawable/ic_edit.xml
Comment (rt4914) [inline app/src/main/res/drawable/ic_edit.xml]: Keep it in multiple of 4, so it should be `24dp`

File: app/src/main/res/drawable/ic_edit.xml
Comment (rt4914) [inline app/src/main/res/drawable/ic_edit.xml]: Ditto

**Review by rt4914 — CHANGES_REQUESTED** [review]
The image and icon does not match what we want. For reference you can see how the profile is show in first screen i.e. `Select your Profile`. I can give you more hints but I think you try to work on it for some more time, if you get stuck then feel free to mention I will guide you to the solution.

**Comment by aggarwalpulkit596** [comment]
> The image and icon does not match what we want. For reference you can see how the profile is show in first screen i.e. `Select your Profile`. I can give you more hints but I think you try to work on it for some more time, if you get stuck then feel free to mention I will guide you to the solution.

Can you explain a little bit about this I couldn't understand what's the use case

**Comment by rt4914** [comment]
> > The image and icon does not match what we want. For reference you can see how the profile is show in first screen i.e. `Select your Profile`. I can give you more hints but I think you try to work on it for some more time, if you get stuck then feel free to mention I will guide you to the solution.
> 
> Can you explain a little bit about this I couldn't understand what's the use case

Currently the image that you have used is not correct in CircularImageView as well as floating action button. It does not match the UI.

Use `ic_default_avatar.xml` instead of `ic_account` (delete this file). Also, try to change the color of floating action button image and background such that it matches the mock. Also, the size of floating action button should be `mini`.

**Comment by aggarwalpulkit596** [comment]
@rt4914 Can I use ShapeableImageView instead of CircleImageView it provides all the same set of functionalities and it's part of Material Library

**Comment by rt4914** [comment]
> @rt4914 Can I use ShapeableImageView instead of CircleImageView it provides all the same set of functionalities and it's part of Material Library

Actually, for now use `CircleImageView` only to have consistency at all places. And also create an issue in this project mentioning to change `CircleImageView` to `ShapeableImageView`, so that we can fix this everywhere in future to provide consistency. You can create new issue using this link: https://github.com/oppia/oppia-android/issues/new

**Comment by aggarwalpulkit596** [comment]
Screenshot_1582750248
Is it good now ?

**Comment by rt4914** [comment]
> Screenshot_1582750248
> Is it good now ?

This looks good, but I don't see any code change, that means that you haven't pushed the code yet. Please push the code and then only I can confirm fully given the case that the screenshot looks good right now.

**Comment by aggarwalpulkit596** [comment]
@rt4914 I've pushed the changes was waiting for your approval just one issue is that in the mock there a little bit elevation to circleImageView but elevation can be applied for v21 and up

**Comment by rt4914** [comment]
> @rt4914 I've pushed the changes was waiting for your approval just one issue is that in the mock there a little bit elevation to circleImageView but elevation can be applied for v21 and up

Thanks for the update. That won't be an issue for now.

File: .idea/codeStyles/Project.xml
Comment (rt4914) [inline .idea/codeStyles/Project.xml]: We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

File: .idea/gradle.xml
Comment (rt4914) [inline .idea/gradle.xml]: We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

File: app/src/main/res/drawable/ic_account.xml
Comment (rt4914) [inline app/src/main/res/drawable/ic_account.xml]: Remove this file, we are no longer using this file.

File: app/src/main/res/layout/add_profile_activity.xml
Comment (rt4914) [inline app/src/main/res/layout/add_profile_activity.xml]: The name for this icon should be `edit_image_fab`

**Comment by aggarwalpulkit596** [comment]
@rt4914 you can now review the PR :)

File: app/src/main/res/layout/add_profile_activity.xml
Comment (rt4914) [inline app/src/main/res/layout/add_profile_activity.xml]: Remove this. And add this:
`app:civ_circle_background_color="@color/grey"`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Some changes suggested.

Also, click on `FloatingActionButton` should result in same thing as `CircleImageView` which currently is missing.

File: .idea/gradle.xml
Comment (rt4914) [reply .idea/gradle.xml]: To fix this You can also, just add blank line at the end.

Comment (aggarwalpulkit596) [reply .idea/gradle.xml]: Can i add this file to git ignore it will happen for every PR

**Comment by aggarwalpulkit596** [comment]
@rt4914 updated

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
